pkg/revocation/db_test.go

```diff
@@ -151,3 +151,38 @@ func TestRevocationDB_Put_error(t *testing.T) {
 		}
 	})
 }
+
+func TestRevocationDB_List(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	testrevocation.RunDBs(t, func(t *testing.T, revDB extensions.RevocationDB, db storage.KeyValueStore) {
+		keys, chain, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
+		require.NoError(t, err)
+		keys2, chain2, err := testpeertls.NewCertChain(2, storj.LatestIDVersion().Number)
+		require.NoError(t, err)
+
+		// test list no revocations, should not error
+		revs, err := revDB.List(ctx)
+		assert.NoError(t, err)
+		assert.Nil(t, revs)
+
+		// list 1,2 revocations
+		firstRevocation, err := extensions.NewRevocationExt(keys[peertls.CAIndex], chain[peertls.LeafIndex])
+		require.NoError(t, err)
+
+		err = revDB.Put(ctx, chain, firstRevocation)
+		require.NoError(t, err)
+		revs, err = revDB.List(ctx)
+		assert.NoError(t, err)
+		assert.Equal(t, 1, len(revs))
+
```

Comment:
It probably wouldn't be the worst thing to compare the actual revocations, just to ensure nothing weird is going on with `list`. To do this you could marshal the `*Revocation` that's returned by `revDB.List` and compare that agains the `Value` of the extension that's returned by `extensions.NewRevocationExt`; like what `TestRevocationDB_Get` is doing:
```
			revBytes, err := rev.Marshal()
			assert.NoError(t, err)
			assert.True(t, bytes.Equal(ext.Value, revBytes))
```